modules/virt-creating-layer2-nad-cli.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_networking/virt-connecting-vm-to-ovn-secondary-network.adoc              
+
+:_content-type: PROCEDURE                                             
+[id="virt-creating-layer2-nad-cli_{context}"]                                   
+= Creating a NAD for flat layer 2 topology using the CLI
+
+You can create a network attachment definition (NAD) which describes how to attach a pod to the layer 2 overlay network.
+
+.Prerequisites
+* You have access to the cluster as a user with `cluster-admin` privileges.
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+. Create a `NetworkAttachmentDefinition` object. The following example configures a cluster-wide overlay without a subnet defined. This means that the logical switch implementing the network only provides layer 2 communication. You must define a static IP address for the virtual machine.
++
+[source,yaml]
+----
+apiVersion: k8s.cni.cncf.io/v1
+kind: NetworkAttachmentDefinition
+metadata:
+  name: l2-network
+  namespace: default
+spec:
+  config: |2
+    {
+            "cniVersion": "0.3.1", <1>
+            "name": "l2-network", <2>
+            "type": "ovn-k8s-cni-overlay", <3>
+            "topology":"layer2", <4>
+            "mtu": 1300, <5>
+            "netAttachDefName": "default/l2-network" <6>
+    }
+----
+<1> The CNI specification version. This value must be less than `1.0.0`.
```

Comment:
Sounds like a bug, I would not bother calling it out here. We don't expect people to change this, they should treat it as a magic number. "This value must be less than `1.0.0`." assumes users know what CNI versions are available. Or just say that the required value is "0.3.1".

Reply:
That is what it said before ([see comment](https://github.com/openshift/openshift-docs/pull/64435#discussion_r1321582610)) :-) Will change it back again.

Reply:
I'm OK with it FWIW, but again ... it can be seen as a bug, or just "we don't support conflist based configurations on this plugin at all".

I guess it's semantics. 

BTW. Why 0.3.1 and not 0.4.0 for instance ?...

Reply:
So, what's the verdict on this one? Should I keep or remove the second sentence in the call out description? Just say `The CNI specification version.`?

Reply:
Use the 0.3.1, as @phoracek suggests. 

Seems it's [the documented value for pods](https://docs.openshift.com/container-platform/4.13/networking/multiple_networks/configuring-additional-network.html#configuration-ovnk-network-plugin-json-object_configuring-additional-network).

Reply:
Sounds good